client/src/components/Menu.js

```diff
@@ -1,11 +1,58 @@
-import React from "react";
-import Grid from "@mui/material/Grid";
+import React, { useState } from "react";
+import Button from "@mui/material/Button";
+import LoadingButton from "@mui/lab/LoadingButton";
+import { Stack, TextField } from "@mui/material";
+import getResults from "../util";
+import { useNavigate } from "react-router-dom";
 
 function Menu() {
+  const navigate = useNavigate();
+  const fetchResults = async () => {
+    let result = null;
+    try {
+      result = await getResults("all", searchItem);
```

Comment:
Here in place of "all", the user can give specific companies as well according to the previous team's documentation.